include/odp/api/spec/init.h

```diff
@@ -317,24 +298,40 @@ int odp_init_local(odp_instance_t instance, odp_thread_type_t thr_type);
  *
  * This function is the second to final ODP call made when terminating
  * an ODP application in a controlled way. It cannot handle exceptional
- * circumstances. In general it calls the API modules per thread terminate
- * functions in the reverse order to that which the module init functions were
- * called during odp_init_local().
+ * circumstances. It is recommended that application frees all ODP resources
```

Comment:
grammar: `that applications free all...`

Reply:
Changed to this: "It is recommended that all ODP resources are freed before the last thread (of the instance) calls this function."

Plural didn't sound good to me as the sentence before is in singular: "...terminating
 an ODP application in a controlled way."